File: clean/clean.go
package clean

import (
	"github.com/aura-studio/dynamic-node-cli/config"
)

// CleanForProcedure executes clean operations for a given procedure and type.
func CleanForProcedure(proc config.Procedure, t CleanType) {
	pl := NewPathListForProcedure(proc)
	c := New(pl)
	c.Clean(t)
}

File: src/clean/clean.js
import { Cleaner } from "./cleaner.js";
import { newPathListForProcedure } from "./path-list.js";

export function cleanForProcedure(proc, type) {
  const pathList = newPathListForProcedure(proc);
  const cleaner = new Cleaner(pathList);
  return cleaner.clean(type);
}
